Benchmarks/Fibonacci/project.json

```diff
@@ -0,0 +1,49 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable",
+    "emitEntryPoint": true,
+    "compilerName": "fsc",
+    "define": [ "TRACE" ],
+    "xmlDoc": true,
+    "compile": {
+      "includeFiles": [
+        "Fibonacci.fs"
+      ]
+    },
+    "additionalArguments": [
+      "--warnon:1182"
+    ]
+  },
+  "runtimeOptions": {
+    "configProperties": {
+      "System.GC.Server": true
+    }
+  },
+  "dependencies": {
+    "FSharp.Core": "4.0.1.7-alpha",
+    "Hopac": { "target": "project" },
+    "Hopac.Platform": { "target": "project" }
+  },
+  "frameworks": {
+    "net46": {},
```

Comment:
Hmm... Does this do more than just make the project to be built for net46?  If so, isn't this unnecessary as there are already separate msbuild files that can be used to build for net45?